scripts/koji/koji_build.py

```diff
@@ -111,9 +111,11 @@ def main():
         urls = []
         for d in git_repos:
             remote, hash = get_repo_and_commit_info(d)
+            if test_build:
+                hash = push_bumped_release(d, test_build)
             urls.append(koji_url(remote, hash))
         command = ['koji', 'chain-build', target] + (' : '.join(urls)).split(' ') +  (['--nowait'] if is_nowait else [])
-        subprocess.check_call(['echo'] + command)
+        print('  '.join(command), flush=True)
```

Comment:
same as previous commit, all such changes should be grouped in a separate commit